### PR TITLE
MWPW-147913: Prevent Users From Uploading When They're Signed In

### PR DESCRIPTION
**Pull request by joaquinrivero**
Resolves: [MWPW-147913](https://jira.corp.adobe.com/browse/MWPW-147913)

Test url:

https://main--dc--adobecom.hlx.page/acrobat/online/compress-pdf?martech=off
https://mwpw-147913--dc--adobecom.hlx.page/acrobat/online/compress-pdf?martech=off

### CODE REVIEW

**Review by Blainegunn — CHANGES_REQUESTED** [review]
looks like we're dispatching `IMS:Ready` twice in this file. 
Perhaps we should consolidate.
Maybe was can do `dropZoneContent.style.pointerEvents = 'none';` with `.css` and then remove it after IMS loads?

**Review by joaquinrivero — COMMENTED** [review]
@Blainegunn I will find another way to get it properly

**Review by saragajic11 — APPROVED** [review]

**Comment by peller** [comment]
❗  this covers a corner case but could degrade performance significantly for all users and may miss new cases for error handling

File: acrobat/scripts/scripts.js [hunk 1]
```diff
@@ -338,6 +338,10 @@ if (IMS_GUEST) {
   };
 }
 
+// Load IMS
+const { default: dcLoadIMS } = await import('./dcIMS.js');
```
Comment (peller) [inline acrobat/scripts/scripts.js:342]: also FYI this import adds delay to the startup, so we generally try to inline code here or make the inlining happen in the edgeworker as needed

**Pull request closed without merging**